packages/expo-sqlite/src/SQLite.types.ts

```diff
@@ -0,0 +1,223 @@
+}
+
+/** 4.4.1 Executing SQL statements
+ */
+export interface SQLTransactionSync {
+  executeSql(sqlStatement: string, args?: Array<any>): SQLResultSet;
+}
+
+/** 4.5 Database query results
+ * The insertId attribute must return the row ID of the row that the SQLResultSet
+ * object's SQL statement inserted into the database, if the statement inserted a row.
+ * If the statement inserted multiple rows, the ID of the last row must be the one returned.
+ * If the statement did not insert a row, then the attribute must instead raise an INVALID_ACCESS_ERR exception.
+ *
+ * The rowsAffected attribute must return the number of rows that were changed by the SQL statement.
+ * If the statement did not affected any rows, then the attribute must return zero.
+ * For "SELECT" statements, this returns zero (querying the database doesn't affect any rows).
+ *
+ * The rows attribute must return a SQLResultSetRowList representing the rows returned,
+ * in the order returned by the database. The same object must be returned each time.
+ * If no rows were returned, then the object will be empty (its length will be zero)
+ */
+export interface SQLResultSet {
+  insertId: number;
+  rowsAffected: number;
+  rows: SQLResultSetRowList;
+}
+
+/** SQLResultSetRowList objects have a length attribute that must return the number of
+ * rows it represents (the number of rows returned by the database). This is the length.
+ * Fetching the length might be expensive, and authors are thus encouraged to avoid using
+ * it (or enumerating over the object, which implicitly uses it) where possible.
+ * The object's supported property indices are the numbers in the range zero to length-1,
+ * unless the length is zero, in which case there are no supported property indices.
+ * The item(index) attribute must return the row with the given index index.
+ * If there is no such row, then the method must return null.
+ *
+ * Each row must be represented by a native ordered dictionary data type. In the
+ * JavaScript binding, this must be Object. Each row object must have one property
+ * (or dictionary entry) per column, with those properties enumerating in the order
+ * that these columns were returned by the database. Each property must have the
+ * name of the column and the value of the cell, as they were returned by the database
+ */
+export interface SQLResultSetRowList {
+  length: number;
+  item(index: number): any;
+}
+
+/** 4.6 Errors and exceptions - asynchronous database API error
+ */
+export declare class SQLError {
+  static UNKNOWN_ERR: number; // = 0;
+  static DATABASE_ERR: number; // = 1;
+  static VERSION_ERR: number; // = 2;
+  static TOO_LARGE_ERR: number; // = 3;
+  static QUOTA_ERR: number; // = 4;
+  static SYNTAX_ERR: number; // = 5;
+  static CONSTRAINT_ERR: number; // = 6;
+  static TIMEOUT_ERR: number; // = 7;
+
+  code: number;
+  message: string;
+}
+
+// synchronous database API error
+export declare class SQLException {
```

Comment:
It's duplicate of above `SQLError` - get rid of it, eventually preserving comments